l1-contracts/contracts/bridge/L1SharedBridge.sol

```diff
@@ -37,14 +37,11 @@ contract L1SharedBridge is IL1SharedBridge, ReentrancyGuard, Initializable, Owna
     IBridgehub public immutable override bridgehub;
 
     /// @dev Legacy bridge smart contract that used to hold ERC20 tokens.
-    IL1ERC20Bridge public immutable override legacyBridge;
+    IL1ERC20Bridge public override legacyBridge;
```

Comment:
Please don't place storage variable between immutables. Harder to undestand the storage layout this way.

Reply:
Fixed in 31436ae176dac6be1685fa25b20578fd4419a1dc